content/english/anti-vegan/no-food-is-fully-vegan.md

```diff
@@ -11,33 +11,103 @@ weight: 5
 
 > It's impossible to have completely vegan food. Fertilizers are derived from animal waste and bones, pesticides are used on crops to kill insects, bees are kept in hives to pollinate almonds, worms are killed by tilling fields. All food involves death.
 
+This can be unpacked as three different arguments:
+
+1. You eat foods that are not completely vegan therefore your claim that animal exploitation is morally wrong is false.
+
+2. You accept the deaths of some animals to produce your foods of preference. This is morally indistinguishable from the deaths that occur to produce my foods of preference.
+
+3. Veganism is logically impossible - it is impossible to produce food without exploiting animals. 
+
 # Rebuttal
 
-## Quick Response
+## Quick Responses
 
-> That's completely correct, it is almost impossible to avoid being complicit in suffering with our current food chain. Veganism is a way of living which seeks to exclude, as far as is possible and practicable, all forms of exploitation of, and cruelty to, animals [...]. Raising an animal from forcible impregnation to their untimely death merely to satisfy our desire for taste is not comparable to deaths we have no way of avoiding as consumers.  
+> (Argument #1) This argument is an appeal to hypocrisy (*the Tu quoque fallacy*). That I fail to meet my own standards is nothing more than a personal failure and does not invalidate my principles. My failure to pick up a piece of litter that I've dropped does not invalidate my claim that littering is wrong. That I may fall short and in some way be complicit in an animal's death in no way invalidates the moral claim that the exploitation of animals is wrong.
+> 
+> (Argument #2) "It is wrong to inflict unnecessary death on nonhuman animals and it is especially wrong, pro tanto, to do so intentionally. Where one must choose between bringing about death as a side-effect, one should, all else equal, pursue the latter course of action".  
```

Comment:
Can we add `[(all other things being equal)]` after pro tanto like Val suggests?

Reply:
No.   Well not really... it's doesn't really capture the meaning.

That said I couldn't come up with anything short that did. I've currently ...'d it out (Didn't want to but it either that or a paragraph of explanation.